src/cmd/cmd.zig

```diff
@@ -808,7 +808,10 @@ fn validator() !void {
 fn shredCollector() !void {
     const allocator = gpa_allocator;
     var app_base = try AppBase.init(allocator);
-    defer app_base.deinit();
+    defer {
```

Comment:
Given the comment here https://github.com/Syndica/sig/blob/82a241843c3d1ba27c898d32f2ac27741f9dabe4/src/cmd/cmd.zig#L1228

Any reason not to call `self.shutdown()` in the `deinit` function?

Reply:
It's critical that `shutdown` is only called once, it's UB to call it twice. I'd rather have the shutdown and deinit happen separately in the right order.